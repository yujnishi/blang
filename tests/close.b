main() {
    auto fd,fname "close.tmp",NULL 0;

    fd = creat( fname, 384 /* 0600 */);
    write(fd,"Hello World.",12);
    printf("1st close return is %d*n",close(fd));
    printf("2nd close return is %d*n",close(fd));
    printf("===*n");

    if ( !fork() ) execl("/bin/cat","cat",fname,NULL);
    wait();
}

