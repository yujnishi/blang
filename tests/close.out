1st close return is 0
2nd close return is -1
===
Hello World.